tests/testthat/test-get_acoustic_detections.R

```diff
@@ -112,22 +117,27 @@ test_that("get_acoustic_detections() allows selecting on animal_project_code", {
   # Select single value
   single_select <- "2014_demer"
   single_select_df <- get_acoustic_detections(con, animal_project_code = single_select)
-  expect_equal(
+  expect_identical(
     single_select_df %>% distinct(animal_project_code) %>% pull(),
     c(single_select)
   )
   expect_gt(nrow(single_select_df), 0)
 
   # Selection is case insensitive
-  expect_equal(
-    get_acoustic_detections(con, animal_project_code = "2014_demer", limit = TRUE),
-    get_acoustic_detections(con, animal_project_code = "2014_DEMER", limit = TRUE)
+  expect_identical(
+    # a limited query doesn't always return the same results, the order isn't guaranteed
+    arrange(
+      get_acoustic_detections(con, animal_project_code = "2014_demer", limit = FALSE),
+      detection_id),
+    arrange(
+      get_acoustic_detections(con, animal_project_code = "2014_DEMER", limit = FALSE),
+      detection_id)
```

Comment:
These two (equivalent) queries each create a 33MB object, we should probably look for `animal_project_code` values that result in smaller objects for testing, that would speed things up.